validations/transfer.test.ts

```diff
@@ -0,0 +1,136 @@
+import { expect, use } from 'chai'
+import { Wallet, ethers } from 'ethers'
+import path from 'path'
+import { randomUUID } from 'crypto'
+import { AddKycParams, FiatConnectClient } from '@fiatconnect/fiatconnect-sdk'
+import {
+  KycSchema,
+  TransferStatus,
+  Network,
+} from '@fiatconnect/fiatconnect-types'
+import { config } from '../src/config'
+import { chaiPlugin } from 'api-contract-validator'
+import { MOCK_QUOTE } from '../src/mock-data/quote'
+import { MOCK_KYC } from '../src/mock-data/kyc'
+import { MOCK_FIAT_ACCOUNTS } from '../src/mock-data/fiat-account'
+
+const apiDefinitionsPath = path.join(config.openapiSpec)
+use(chaiPlugin({ apiDefinitionsPath }))
+
+describe('/transfer', () => {
+  const mockAccountData =
+    MOCK_FIAT_ACCOUNTS[
+    config.fiatAccountMock as keyof typeof MOCK_FIAT_ACCOUNTS
+  ]
+
+  const mockKYCInfo: AddKycParams<KycSchema> =
+    MOCK_KYC[config.kycMock as keyof typeof MOCK_KYC]
+
+  describe('/in', () => {
+    const wallet = Wallet.createRandom()
```

Comment:
shouldn't this use config.testPrivateKey?

Reply:
It probably should just to stay consistent with our other tests, but I personally like the idea of using a random wallet/address on each test. That way, we hopefully shouldn't have to worry about leftover state relating to a single wallet on the provider's side when running subsequent tests.

Reply:
Since these tests _should_ pass while using a random wallet address, I'm fine with keeping this as `Wallet.createRandom`. Perhaps in a follow-up PR we can change our current uses of `config.testPrivateKey` to using a random wallet, where applicable (non-transfer tests).